app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/NavigationOverlayFragment.kt

```diff
@@ -72,14 +71,13 @@ private const val COL_COUNT = 5
 private val uiHandler = Handler(Looper.getMainLooper())
 
 enum class NavigationEvent {
-    SETTINGS, BACK, FORWARD, RELOAD, LOAD_URL, LOAD_TILE, TURBO, PIN_ACTION, POCKET, DESKTOP_MODE, EXIT_FIREFOX;
+    BACK, FORWARD, RELOAD, LOAD_URL, LOAD_TILE, TURBO, PIN_ACTION, POCKET, DESKTOP_MODE, EXIT_FIREFOX;
```

Comment:
Hm, when we hit 'send usage data' or 'clear all data' it opens a new screen.  The screen is very simple (it can only go back to the overlay), but is this something we should consider a navigation event?

This might be something that we add in when it becomes a problem, but I'm wondering if this makes our navigation code less consistent and therefore harder to follow.

Reply:
Makes sense - it's a little awkward now because the SettingsType now gets converted into a NavigationEvent, and then back to a SettingsType for ScreenController, but putting it into the main flow of NavigationEvent means there aren't multiple places to look for navigation, which is good - and we can also clean it up later.